### PR TITLE
Feature/liwo 451

### PR DESCRIPTION
**Pull request by SiggyF**
- Replace print server with brower based printing
- Prints background map
- Prints GEE layers.

### CODE REVIEW

File: src/components/ExportPopup.vue [hunk 1]
```diff
@@ -22,7 +22,8 @@
             <span>Zip</span>
           </label>
         </li>
-        <li class="choice-cards__item">
+        <!-- disable for now -->
```
Comment (CindyvdVries) [inline src/components/ExportPopup.vue:25]: why?

File: src/components/LiwoMap.vue [hunk 2]
```diff
@@ -1,20 +1,30 @@
...
+
 </template>
 
 <script>
+
 import createMapConfig from '@/lib/leaflet-utils/mapconfig-factory'
+import { legendControl } from '@/lib/leaflet-utils/legend'
 import { EPSG_28992 } from '@/lib/leaflet-utils/projections'
 
+console.log('legendControl', legendControl)
```
Comment (CindyvdVries) [inline src/components/LiwoMap.vue:27]: remove?

File: src/lib/export-map-image.js [hunk 3]
```diff
@@ -15,28 +15,28 @@ export default async function requestImage (options) {
   let services = await mapConfig.getServices()
   const printGeoServerURI = services.PRINT_GEOSERVER_URL
 
-  return fetch(`${printGeoServerURI}/print/liwo/report.${options.outputFormat}`, {
+  let response = await fetch(`${printGeoServerURI}/print/liwo/report.${options.outputFormat}`, {
```
Comment (CindyvdVries) [inline src/lib/export-map-image.js:18]: why async?
no catching errors anymore?

File: src/lib/leaflet-utils/legend.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import L from '@/lib/leaflet-utils/leaf'
+
+const LegendControl = L.Control.extend({
+  onAdd: function (map) {
+    var div = L.DomUtil.create('div', 'info legend')
+    // make a copy  of hte legend and add it to the print map
```
Comment (CindyvdVries) [inline src/lib/leaflet-utils/legend.js:6]: the :)

**Comment by CindyvdVries** [comment]
add tests?

File: src/lib/export-map-image.js (hunk 3 above)
Comment (SiggyF) [reply src/lib/export-map-image.js]: removed file alltogether.

File: src/components/ExportPopup.vue (hunk 1 above)
Comment (SiggyF) [reply src/components/ExportPopup.vue:25]: Backend is not yet implemented.

**Pull request merged**